messaging/coap/engine.c

```diff
@@ -873,7 +890,7 @@ coap_receive_request_with_code(coap_receive_ctx_t *ctx, oc_endpoint_t *endpoint)
       uint32_t buffer_size = (uint32_t)OC_MAX_APP_DATA_SIZE;
       ctx->response_buffer = oc_blockwise_alloc_response_buffer(
         oc_string(client_cb->uri) + 1, oc_string_len(client_cb->uri) - 1,
-        endpoint, client_cb->method, OC_BLOCKWISE_CLIENT, buffer_size);
+        endpoint, client_cb->method, OC_BLOCKWISE_CLIENT, buffer_size, false);
```

Comment:
Do we want generate response etag in some cases?

Reply:
Yes, but only when we have a block wise message with multiple blocks, that's how it worked previously.